testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -426,6 +426,30 @@ def forward(self, x):
 output = np.mean(input, axis=(3, 4), keepdims=True)
 save_onnx_data_and_model(input, output, 'reduce_mean3d', 'ReduceMean', axes=(3, 4), keepdims=True)
 
+class Split(nn.Module):
+
+    def __init__(self, *args, **kwargs):
+        super(Split, self).__init__()
+        self.split_size_sections = \
+            kwargs.get('split_size_sections', 2)
+        self.dim = kwargs.get('dim', 0)
+
+    def forward(self, x):
+        return torch.split(x, self.split_size_sections, self.dim)
+
+model = Split()
+input = Variable(torch.randn([1., 2., 3., 4., 5., 6.]))
+save_data_and_model("split_1", input, model)
+
+model = Split(dim=0)
+save_data_and_model("split_2", input, model)
+
+model = Split(split_size_sections=[2, 4])
+save_data_and_model("split_3", input, model)
+
+model = Split(dim=0, split_size_sections=[2, 4])
+save_data_and_model("split_4", input, model)
```

Comment:
Looks like files are missed.

Reply:
Yes, in the latest logs, `findData` is unable to find the files.

Reply:
So this PR should include these files

Reply:
Actually, `onnx.checker.check_model(model_def)` is throwing `Segmentation Fault` on my system. Probably that's why code never reaches to the above lines and the already present files are detected but new ones aren't generated. I will add my `onnx` files separately and will push here.

Reply:
> So this PR should include these files

Done. Please take a look.